### ISSUE TITLE
Vulnerability Management for packages

### ISSUE THREAD

**Issue by afsahsyeda** [eclipse-sw360/sw360#2093]
[//]: # (Copyright Siemens AG, 2021. Part of the SW360 Portal Project)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

### Description

Vulnerabilities should be tracked at the package level.

#### Steps followed and expected result

- Create a Vulnerabilities tab on the package details page for tracking each package
- At the project level, currently, only release ids are sent for monitoring. After this feature, package ids should be sent instead of the release id they are linked to. If a release does not have any linked packages in the project, send its release id itself.

**Comment by KoukiHama** [comment]
Very Good Idea.
Do you have plans for implementation or just ideas?

**Comment by afsahsyeda** [comment]
Yes, a plan has been formulated and the work should start next week
